ios/Classes/SwiftAdmobFlutterPlugin.swift

```diff
@@ -1,14 +1,50 @@
-import Flutter
+/*
+ ISC License
+ 
+ Copyright (c) 2019 Kevin McGill <kevin@mcgilldevtech.com>
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
 import UIKit
+import GoogleMobileAds
 
 public class SwiftAdmobFlutterPlugin: NSObject, FlutterPlugin {
+    
   public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "admob_flutter", binaryMessenger: registrar.messenger())
     let instance = SwiftAdmobFlutterPlugin()
-    registrar.addMethodCallDelegate(instance, channel: channel)
+    
+    let defaultChannel = FlutterMethodChannel(name: "admob_flutter", binaryMessenger: registrar.messenger())
+    registrar.addMethodCallDelegate(instance, channel: defaultChannel)
+    
+    registrar.register(
+        AdmobBannerFactory(messeneger: registrar.messenger()),
+        withId: "admob_flutter/banner"
+    )
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-    result("iOS " + UIDevice.current.systemVersion)
+    switch call.method {
+    case "initialize":
+        GADMobileAds.sharedInstance().start { (initializationStatus: GADInitializationStatus) in
+            print("initializationStatus: \(initializationStatus)")
+        }
+        break
+    case "getPlatformVersion":
+        result("iOS " + UIDevice.current.systemVersion)
+        break
```

Comment:
@YoussefKababe I don't believe this is used. I'm guessing it's left over from boilerplate templates. If so, I will remove it.

Reply:
Yes, that's certainly left over from the examples. We can remove it.

Reply:
I'll remove it in a minute. I'm actually pulling in my fork to an app of mine to test things out right now!